CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to the BCI-dockerfile-generator
+
+##  Opening an Pull Request
+
+The BCI-dockerfile-generator documentation has a section on contributing a
+new container or modifying an existing container. Please check the chapter
+**Adding and modifying containers** for details.
+
+Before creating a pull request, please format your source code with Ruff,
+which is installed in the Poetry virtual environment by default:
+
+```bash
+poetry run ruff format
+# reorder imports:
+poetry run ruff check --fix
+```
+
+Additionally, run the unit tests and check whether the documentation builds
+(additional points if you update it):
+
+```bash
+# tests
+poetry run pytest -vv
+# docs
+poetry run sphinx-build -M html source build -W
+```
+
+
+## Reviewing a Pull Request
+
+The following guidelines are respected by the maintainers on reviewing a Pull Request:
+
+* CI failures are treated seriously. As a general rule changes with failing CI are not being merged.
+
+* A reviewer who has requested a change should be notified when the feedback
+has been handled. This can be done by clicking the "re-request review" button
+on the particular reviewers handle.
+
+* A pull request is considered mergeable if it has no outstanding change
+requests, at least one approval by the maintainers and sufficient time has
+passed. Sufficient time means a business workday since initial creation.
```

Comment:
I would like to extend it to two to three business days. There's busy days and when multiple people decide to send in PRs, I simply don't have the time to review them all.

Reply:
Added the "Dan exception clause".